### PR TITLE
fix(website): Animate sliding customer logos, fix action link styling

### PR DESCRIPTION
**Pull request by Patticatti**

### CODE REVIEW

File: website/src/components/CustomerLogos/index.tsx [hunk 1]
```diff
@@ -9,85 +9,131 @@ export function CustomerLogosGrayscale() {
...
+            height={75}
+          />
+        </Link>
+        <Link className="h-12 mr-20 place-content-center" href="https://teracloud.com">
+          <Image
+            alt="teracloud logo"
+            src="/images/logos/cust-logo-teracloud-gray.svg"
+            width={125}
+            height={125}
+            className="  "
```
Comment (jamilbk) [inline website/src/components/CustomerLogos/index.tsx]:
```suggestion
```

**Review by jamilbk — CHANGES_REQUESTED** [review]

**Comment by jamilbk** [comment]
Looks like CI is failing for the `globals.css` file:

https://github.com/firezone/firezone/actions/runs/10409951593/job/28830709460?pr=6309#step:7:43

Needs a `prettier` run on it.

**Comment by jamilbk** [comment]
- I think some of the logos are cutoff now
- The space between the last logo and first logo is not consistent
- Any chance this could be slowed down by at least half?

Screenshot 2024-08-15 at 2 13 30 PM
Screenshot 2024-08-15 at 2 13 15 PM

**Comment by Patticatti** [comment]
> * I think some of the logos are cutoff now
> * The space between the last logo and first logo is not consistent
> * Any chance this could be slowed down by at least half?
> 
> Screenshot 2024-08-15 at 2 13 30 PM Screenshot 2024-08-15 at 2 13 15 PM

Fixed and slowed down a tad, like to 14s

**Review by jamilbk — APPROVED** [review]
LGTM. Looks like CI failure is preventing merge:

https://github.com/firezone/firezone/actions/runs/10466025623/job/28982194439?pr=6309#step:7:43

**Comment by jamilbk** [comment]
@Patticatti I'll fix the issues and conflicts here to get this merged.

**Pull request merged**